wiki/Article_Style_Guide/en.md

```diff
@@ -499,31 +589,74 @@ Use of this type of link may include:
+Do **not** use the word `here` as the link text.
+As it may seem convenient to use, this may cause confusion and/or be misleading.
+
+Instead, take the page's title (from the top of page or from the `<title/>` tag) and use that as the link text.
+If there is not a viable or obvious title, then use the site's name or a genernal description instead.
 
 ### Section Links
 
-All section links follow these rules:
+All section links follow these strict rules:
 
 - all letters **must** use lowercase letters
 - all special characters (e.g. `!`, `?`, `,`, `"`, `'`) **must** be replaced with a hyphen (`-`)
   - if one of the spacial characters is the last character in the section name, remove it, but do not add a hyphen (`-`).
 - all spaces are replaced with hyphens (`-`), not underscores (`_`)
-- if a header is not unique (two or more of the same section heading names exist at any level),
+- if a header is not unique (two or more of the same section heading names exist at any level):
   - the first instance of said heading will follow the above rules
-  - the following instances will follow the above rules **and** must add `.` followed by an incremental integer starting from 2
+  - the following instances **must** follow the above rules **and** must add `.` followed by an incremental integer starting from `2`
+- if a section contains an image, ignore it
+- if a section starts with a space, ignore it
 
-You will need to check to make sure it links to the correct section.
+Notice: you will not be able to check if section link links to the correct location in some scenarios.
 
 Examples:
 
 - `[Scoring](#scoring)`
 - `[osu!mania](#osu-mania.2)`
-- `[What is osu!taiko?](#what-is-osu-taiko)`
+- `[What is osu!taiko?](/wiki/FAQ/#what-is-osu-taiko)`
+
+#### Icons in Headings
+
+When an icon is inside the heading, ignore it.
+If there is a space due to the icon, ignore it.
+
+The anchor of a heading with an icon will start with a letter, not a space or hyphen.
+
+### User Profiles
 
-Section linking **will** work for unsafe ASCII characters; however, do know that the section link in the url will be using [percent-encoding](https://en.wikipedia.org/wiki/Percent-encoding).
+When linking to a osu!user profile, use the user's id number instead of their username.
+Usernames can be changed, while user ids are bound to the user.
+This is to ensure that the link still links to the same person even after they changed their username.
+
+Do not:
+
+- `[osuplayer111](https://osu.ppy.sh/u/osuplayer111)`
+- `[Agent Spin Here](https://osu.ppy.sh/u/Agent_Spin_Here)`
+
+Do:
+
+- `[osuplayer111](https://osu.ppy.sh/u/33599 "Andrea")`
+- `[Agent Spin Here](https://osu.ppy.sh/u/41481 "Mashley")`
+
+or
+
+- `[Andrea](https://osu.ppy.sh/u/33599 "Andrea")`
+- `[Mashley](https://osu.ppy.sh/u/41481 "Mashley")`
+
+Unfortunately, osu! doesn't provide an easy, straight-forward way to get the user's id number.
```

Comment:
`Unfortunately, osu! does not provide an easy, straight-forward way to get the user's id number.`